README.md

```diff
@@ -5,8 +5,6 @@
 <p align="center">
 <a href="https://github.com/open-horizon/open-horizon.github.io" alt="GitHub contributors">
 <img src="https://img.shields.io/github/contributors/open-horizon/open-horizon.github.io.svg"/></a>
-<a href="https://github.com/open-horizon/open-horizon.github.io" alt="anazdocscopy.yml">
```

Comment:
Everything else LGTM, but we can skip this change as it is already addressed in PR #405, merging that PR before this PR might cause merge conflict, we can either skip this change in this PR or we can just close the previous PR i.e (#405) and link both issues to this single PR.